plugins/inputs/beat/README.md

```diff
@@ -0,0 +1,149 @@
+# Beat Plugin
+The Beat plugin will collect metrics from the given Beat instances. It is
+known to work with Filebeat and Kafkabeat.
+### Configuration:
+```toml
+  ## An URL from which to read beat-formatted JSON
+  ## Default is "http://127.0.0.1:5066".
+  url = "http://127.0.0.1:5066"
+
+  ## Enable collection of the generic Beat stats
+  collect_beat_stats = true
+
+  ## Enable the collection if Libbeat stats
+  collect_libbeat_stats = true
+
+  ## Enable the collection of OS level stats
+  collect_system_stats = false
+
+  ## Enable the collection of Filebeat stats
+  collect_filebeat_stats = true
+
+  ## HTTP method
+  # method = "GET"
+
+  ## Optional HTTP headers
+  # headers = {"X-Special-Header" = "Special-Value"}
+
+  ## Override HTTP "Host" header
+  # host_header = "logstash.example.com"
+
+  ## Timeout for HTTP requests
+  timeout = "5s"
```

Comment:
You might also want to comment the default to match `sampleConfig`.

Reply:
good catch, will fix.